package.json

```
@@ -11,9 +11,9 @@
   "scripts": {
     "build": "tsc --project .",
     "lint": "eslint . --ext .ts,.js",
-    "test": "npm run build && node test/index.js",
-    "test:browser": "npm run build && browserify test/index.js > test/bundle.js && echo ';window.close();' >> test/bundle.js && cat test/bundle.js | smokestack",
-    "prepublishOnly": "npm run build"
+    "test": "yarn run build && node test/index.js",
+    "test:browser": "yarn run build && browserify test/index.js > test/bundle.js && echo ';window.close();' >> test/bundle.js && cat test/bundle.js | smokestack",
+    "prepublishOnly": "yarn run build"
```

Nit: The "run" isn't required anymore with Yarn:
```suggestion
    "test": "yarn build && node test/index.js",
    "test:browser": "yarn build && browserify test/index.js > test/bundle.js && echo ';window.close();' >> test/bundle.js && cat test/bundle.js | smokestack",
    "prepublishOnly": "yarn build"
```